Components/Transform/Drivers/MultiValueArithmeticDriver.cs

```diff
@@ -1,70 +1,69 @@
 ﻿using System.Linq;
-using Elements.Assets;
 using Elements.Core;
 using FrooxEngine;
 
 namespace Obsidian;
 
+public enum ArithmeticMode
```

Comment:
so enums have to be outside now ?

Reply:
Just needs to be outside of the generic class. So I just dropped it into the `Obsidian` namespace for now. Idk if you want to put it somewhere else though like maybe into its own namespace or something.